dipy/align/tests/test_metrics.py

```diff
@@ -234,7 +250,7 @@ def test_em_demons_step_3d():
 
 
 if __name__ == '__main__':
-    test_em_demons_step_2d()
-    test_em_demons_step_3d()
+    # test_em_demons_step_2d()
+    # test_em_demons_step_3d()
     test_exceptions()
-    test_EMMetric_image_dynamics()
+    # test_EMMetric_image_dynamics()
```

Comment:
Why have these other tests been disabled? Is this left over from when you were debugging?

Reply:
left over, I will correct that...